src/utils/handleFilteredTodos.tsx

```diff
@@ -0,0 +1,17 @@
+import { Filters, Todo } from '../types';
+
+export const handleFilteredTodos = (todos: Todo[], filter: Filters) => {
+  const filtrTodos = [...todos];
```

Comment:
```suggestion
  const filteredTodos = [...todos];
```
Try to give full names